core/modules/wsched/testSchedulers.cc

```diff
@@ -428,6 +428,10 @@ struct SchedFixture {
                 std::chrono::seconds(_examineAllSleep), 5);
         blend = std::make_shared<wsched::BlendScheduler>("blendSched", queries, maxThreads,
                 group, scanSlow, scanSchedulers);
+        group->setPosition(0);
```

Comment:
Why should schedules know about their positions in the list of schedulers? This may result in a convoluted logic because usually elements of a collection should not be aware about any details of their host collections.

Reply:
Position is a form of priority, the position kept within the scheduler is its default position. If the position is changed in the list for temporary conditions, the default position allows for it to be restored easily. I'll change the name for clarity.